src/main/webapp/index.html

```diff
@@ -2,10 +2,42 @@
 <html>
     <head>
```

Comment:
Can you rename this file from `index.html` to `createevent.html`? Or something along those lines? Just want the file name to represent the contents (index.html is usually for the home page)